### DISCUSSION TITLE
Beta 13 hangs during startup

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#1032] [Help]
So it seems that b13 is hanging while scanning some plugins.

What i can guess so far : 
- AG hangs on plugins that are there and unchanged since beta 6
- Those plugins are constantly scanned at startup because of validations issues with AG (when they work fine in DAW)
- Before beta 13 AG never hangs while scanning. 

Right now it is stuck for an hour on this one ...

**Comment by Henri75** [comment]
![Screen Shot 2022-09-18 at 15 44 06](https://user-images.githubusercontent.com/56123216/190906743-3f6b30f9-29e5-4c31-890a-d4046bd637df.png)

**Comment by Henri75** [comment]
tried several times, even removing that plugin but same things happens on other

[15:36:48.470|server|1410465f0] reading scan crash file /Users/me/.audiogridder/audiogridderserver1003.crash

Guess it means the server crashes :(

but the file doesnt exists

**Reply by apohl79** [reply]
If you remove the .crash file(s) and rescan, does that help? A crash file stores info about a plugin that was previously scanned. If it crashes the scanner, it gets blacklisted. But I have never seen the server being stuck reading it.
